src/app/lib/utilities/getRemoteConfig/index.test.js

```diff
@@ -0,0 +1,104 @@
+import fetchMock from 'fetch-mock';
+import { CONFIG_REQUEST_RECEIVED, CONFIG_FETCH_ERROR } from '#lib/logger.const';
+
+const mockUrl =
+  'https://mock-config-endpoint?application=simorgh&service=mundo&__amp_source_origin=http://localhost';
+const mockResponse = { testToggle: { enabled: true } };
+fetchMock.mock(mockUrl, mockResponse);
+
+describe('getRemoteConfig', () => {
+  beforeEach(async () => {
+    jest.resetModules();
+    await import('#testHelpers/loggerMock');
+    process.env.SIMORGH_CONFIG_URL = 'https://mock-config-endpoint';
+  });
+
+  it('should return null if enableFetchingToggles is not enabled', async () => {
+    jest.mock('#lib/config/toggles', () => ({
+      local: {
+        enableFetchingToggles: { enabled: false },
+      },
+    }));
+
+    // Dynamic import is used in these tests so the toggles file values can be changed
+    const getRemoteConfig = await import('.');
+    const remoteToggles = await getRemoteConfig.default('mundo');
```

Comment:
Whatever is passed via `export default` winds up on the default key when using dynamic import. I had to use dynamic import to change the values within the toggles file. It may be better to change the toggles file to return a function rather than an object, so this can be more easily mocked. But I decided to leave that refactor out of this PR since it's getting fairly large.